### DISCUSSION TITLE
Bounty on tasks created by repository owners

### DISCUSSION THREAD

**Discussion by muratcanoguzhan** [community/community#28107] [Other Feature Feedback, Questions, & Ideas]
I have an idea on Github. For example, I have a repository and I can give a bounty to an issue(or a task created by me) for example if someone is going to solve this and it is accepted by 3 senior-like developers(I even have some cool abbreviations like POD-Proof-Of-Developer concept and even the developer who develop and couldn't get approve can have the chance. I'll tell this also later on ) she/he is gonna get that bounty and so on... and I have an idea like this to get profit for Github and Developers. Developers could earn and projects get developed without companies or with companies just going over Github. I know this is a huge development and I would like to take part in.

**Comment by jfullstackdev** [comment]
I really like the idea that GitHub will give even a small amount of money for developers that are not so popular but the code is still useful, say 100 dollars will do, maybe alongside a certain badge, when it is achieved, like the Artic badge or maybe something similar in the future, that will make me feel so satisfied even with just that small amount

anyways, GitHub just recently gave amount to large-scale projects that they selected but I cannot find the link to that

**Reply by muratcanoguzhan** [reply]
> I really like the idea that GitHub will give even a small amount of money for developers that are not so popular but the code is still useful, say 100 dollars will do, maybe alongside a certain badge, when it is achieved, like the Artic badge or maybe something similar in the future, that will make me feel so satisfied even with just that small amount
> 
> anyways, GitHub just recently gave amount to large-scale projects that they selected but I cannot find the link to that

Yes, a certain badge would be useful otherwise a new member can manipulate the projects in the wrong way. Also when a task is taken and when that task is left should be tracked and maybe this should affect the badge, when someone thinks to give a task to a developer these records should be visible to her/him because this provides how reliable that developer.

**Comment by Salman-dott** [comment]
I read that many people like open-source code applications, but can't figure out why. When I asked my colleague, who owns a calories calculator app, to disclose a code to me for training to make my own healthy food develry app, he got really upset. He said I must [visit website](https://rubyroidlabs.com/blog/2021/10/food-delivery-app-development-from-scratch/) and have my own app or code developed. That his app was exclusively his property. Is he right?

**Comment by hope44441** [comment]
Good afternoon, maybe a little off topic, but I’m looking for a developer for e-commerce for my project, I found it here https://expertremote.io/hire-symfony-developers/, but I don’t know how to find out in person whether it’s a pro or not, tell me
